scripts/generate-CHANGES.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python
```

Comment:
Is this script really python2 and 3 compatible (and correct according to [pep-394](https://www.python.org/dev/peps/pep-0394/))? It looks like it might be.